docs/source/intervention_models/maternal_supplementation/index.rst

```diff
@@ -107,48 +111,66 @@ Given the low utilization of MMS and BEP relative to IFA, we assume that baselin
     - Note
   * - Ethiopia
     - Pregnant population
-    - Percent who took antenatal iron for 90+ days
-    - 10.6
+    - Proportion who took *any* antenatal iron
+    - 0.598 (0.583, 0.613), normal distribution of uncertainty clipped between zero and one
+    - Use this value for the :ref:`acute malnutrition simulation <2019_concept_model_vivarium_ciff_sam>`; DHS 2019
+  * - Ethiopia
+    - Pregnant population
+    - Proportion who took antenatal iron for 90+ days
+    - 0.106
```

Comment:
Is uncertainty measurement available for other countries?

Reply:
I added a note about how this can be done for the other countries using the reported sample size from DHS, but haven't actually gone through and pulled the numbers because I don't think that we are planning to use them in the near future.